tests/integration/factory-test.js

```diff
@@ -77,4 +82,43 @@ test('db.admins({secret: "secret123"})', function (t) {
   })
 
   .catch(t.error)
+
+  .then(function () {
+    return new Promise(function (resolve, reject) {
+      var salt = generateSalt()
+      hashPassword('mysecret', salt, 10, function (error, hash) {
+        if (error) {
+          return reject(error)
+        }
+
+        return resolve(format(
+          '-%s-%s,%s,%s',
+          'pbkdf2',
+          hash,
+          salt,
+          10
+        ))
+      })
+    })
+  })
+  .then(function (userEntry) {
+    preinitializedAdmins = db.admins({secret: 'secretXYZ', admins: {'foo': userEntry}})
```

Comment:
Yeah, no need to run `generateSalt` or `hashPassword`, it was just for convenience rather than hardcoding in a pre-generated key at the time; but it probably makes sense to pregenerate it. Changing now.